### ISSUE TITLE
cc1: all warnings being treated as errors

### ISSUE THREAD

**Issue by gitrubus** [dnschneid/crouton#4659]
```
Please paste the output of the following command here: name: mojar
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/mojar...
crouton: version 1-20220107165406~master:811f5e45
release: xenial
architecture: amd64
targets: unity
host: version 14268.67.0 (Official Build) stable-channel hatch 
kernel: Linux localhost 4.19.208-12427-gf85a69d601ba #1 SMP PREEMPT Sun Dec 12 19:53:49 PST 2021 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
Unmounting /mnt/stateful_partition/crouton/chroots/mojar...
name: xenial
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/xenial...
crouton: version 1-20220107165406~master:811f5e45
release: xenial
architecture: amd64
targets: unity
host: version 14268.67.0 (Official Build) stable-channel hatch 
kernel: Linux localhost 4.19.208-12427-gf85a69d601ba #1 SMP PREEMPT Sun Dec 12 19:53:49 PST 2021 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
Unmounting /mnt/stateful_partition/crouton/chroots/xenial...
```

**Comment by gitrubus** [comment]
>cc1: error: -Werror=stringop-overread: no option -Wstringop-overread
cc1: error: unrecognized command line option '-Wno-int-in-bool-context' [-Werror]
cc1: all warnings being treated as errors

**Comment by myelin** [comment]
Sorry, this is a bug I introduced in #4658.  It'll be fixed by #4662.
